styles.css

```diff
@@ -280,3 +307,84 @@ li {
 .myUL-skills li {
   justify-content: center;
 }
+
+.contact-form {
+  padding: 120px 24px 120px 24px;
+}
+
+.contact-me {
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  color: #172b4d;
+  font-size: 32px;
+  font-style: bold;
+  text-align: center;
+}
+
+.name-field,
+.email-field {
+  border: 1px solid;
+  border-color: #ded8d7;
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  margin-top: 50px;
+  width: 98%;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  color: #1a1919;
+  padding-left: 10px;
+}
+
+.text-area-field {
+  width: 98%;
+  background-color: #fbf8f7;
+  margin-top: 20px;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  font-style: italic;
+  border: none;
+  padding-top: 10px;
+  padding-left: 10px;
+}
+
+.get {
+  display: flex;
+  justify-content: center;
+}
+
+.get-in-touch {
+  margin-top: 50px;
+  padding: 12px;
+  background-color: #ff6b00;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  color: #fff;
+  border: none;
+}
+
+.horizontal {
+  border: 1px solid #dfe1e6;
+  margin-top: -40px;
+}
+
+.social-icon ul {
+  display: flex;
```

Comment:
class definition needs improvements

Reply:
- [ ] Good work using GitHub features to perform this review 👏🏾. Please find my suggestions below on how to improve this review. 
 - Please always give clear and easy-to-understand comments. You can go ahead and state what kind of improvement the class definition needs. For example "**I think your class names are not very descriptive**". 👍🏾 
- Please also remember to include why a change or r recommendation is necessary. For example "**Descriptive class names make it easy to understand your code**" ✔️ 
- Please always remember to give an example when necessary. This helps the student to understand what exactly needs to be worked on. 😀